src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. **Use of Exception for Testing**:
   - The comment in the `throwError` method mentions that the exception is deliberately thrown for PR review. While this might be okay for testing purposes, it's important to ensure that such test code is removed before deploying to production.